.github/workflows/build.yml

```diff
@@ -42,7 +47,7 @@ jobs:
       max-parallel: 4
       matrix:
         # test against latest update of each major Java version, as well as specific updates of LTS versions:
-        java: [8, 8.0.192, 11.0.x, 11.0.3, 12, 13, 15, 16, 17 ]
```

Comment:
Having trouble with java 8 with the examples build. I will put this back in once i've fixed the issue.